var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -125,6 +120,11 @@ def cmake_args(self):
             'ON' if '+cuda' in spec else 'OFF'
         ))
 
+        # Tests
+        args.append('-DHPX_WITH_TESTS={0}'.format(
+            'ON' if '+tests' in spec else 'OFF'
```

Comment:
Does this change what gets installed? If not, I would do:
```suggestion
            'ON' if self.run_tests else 'OFF'
```
That way we don't add an unnecessary variant that doesn't change what gets installed. Tests can be activated with `spack install --test=root hpx`.